scripts/AbinsModules/LoadCRYSTAL.py

```diff
@@ -529,6 +485,7 @@ def create_kpoints_data_helper(self, atomic_displacements=None, atomic_coordinat
         :param column: number of atomic_displacements column to parse
         :param freq_num: number of mode (frequency)
         :param row_width: current width of row to parse
+        :return normalised atomic displacements
         """
         xdisp = atomic_displacements[0]
         ydisp = atomic_displacements[1]
```

Comment:
In the loop starting on Line 498, there is a lot recalculated in every iteration, including the number of atomic_coordinates

Reply:
Here atomic displacements are rescaled. I can rearrange code so it will be more compact.